Bugzilla/BugMail.pm

```diff
@@ -595,7 +595,7 @@ sub _get_diffs {
       $diff->{num}       = $comment->count;
       $diff->{isprivate} = $diff->{new};
     }
-    elsif ($diff->{field_name} eq 'dependson' || $diff->{field_name} eq 'blocked') {
+    elsif ($diff->{field_name} =~ /^(dependson|blocked|regress(ed_by|es))$/) {
```

Comment:
```suggestion
    elsif ($diff->{field_name} =~ /^(?:dependson|blocked|regress(ed_by|es))$/) {
```
Use a non-capture group since you don't care about the captured pattern